packages/animation/src/utils/getGlobalSpace.js

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * @typedef {import('../../edit-story/types').Element} Element
+ * @typedef {import('../types').StoryElementPropType} Element
```

Comment:
It's not possible to import a prop type (an actual JS object) as a a JSDoc type.

We need to import the actual JSDoc type definition.

That means copying this piece of code:

https://github.com/google/web-stories-wp/blob/270cc1726bce0e25c0ca8e551ed13e66202dc232/assets/src/edit-story/types.js#L245-L262

to `packages/animation/src/types.js` and changing this line here accordingly.

```suggestion
 * @typedef {import('../types').Element} Element
```